os/include/tinyara/fs/fs_tmpfs.h

```diff
@@ -0,0 +1,228 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
This is already included here right ?  os/fs/tmpfs/fs_tmpfs.h  
Why do we need to include at 2 places ?

Reply:
okay will remove it